src/Stan/Observation.hs

```diff
@@ -33,7 +34,7 @@ import Stan.Core.Toggle (isHidden)
 import Stan.Hie.Debug ()
 import Stan.Inspection (Inspection (..))
 import Stan.Inspection.All (getInspectionById)
-import Stan.Report (ReportSettings (..))
+import Stan.Report.Settings (ReportSettings (..))
```

Comment:
Very nice idea to move `ReportSettings` into the separate module! This will help with incremental compilation a lot 👍